SR2019/2018-10-10-place-confirmation.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019
+subject: Confirming your place at Student Robotics 2019
+note: This email will be sent multiple times, to those in the list not in the "Contacted" segment
+---
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2019! If you've applied for a second team, we sadly can't confirm you will receive a second kit until our deadline on the 26th.
+
+The specific details for the London kickstart will be sent over soon to those who have requested the London venue. The London venue is small, and so we're only able to handle a small number of teams. Please only go to this venue if you've been confirmed. The Southampton venue (University of Southampton) will be available to all other teams.
```

Comment:
Maybe use the templating in MailChimp to make this paragraph different based on where we want teams to go?

Reply:
Perhaps, but I think i'd rather take the easy option and just give the information to everyone. Just so everyone has all the information, especially as we're going to have to limit the number of teams anyway